elixir-sdk/.cargo/config.toml.template

```diff
@@ -0,0 +1,11 @@
+# Copy this file to .cargo/config.toml (without the .template extension)
+# to use the local eppo_core package during development.
+
+[target.'cfg(target_os = "macos")']
+rustflags = [
+    "-C", "link-arg=-undefined",
+    "-C", "link-arg=dynamic_lookup",
+]
+
+[patch.crates-io]
+eppo_core = { path = "../../eppo_core" } 
```

Comment:
this overrides the import of eppo_core to use the local package

Reply:
isn't this too much `..`? I believe it's counting starting from `elixir-sdk`
```suggestion
eppo_core = { path = "../eppo_core" } 
```

Reply:
Yes good eyes! I am certain I fixed this before but clearly not 🤔  -- updated.